terracotta/image.py

```diff
@@ -86,9 +88,10 @@ def array_to_png(img_data: Array,
                     alpha,
                     np.zeros(256 - len(colormap) - 1, dtype='uint8')
                 ))
-                assert transparency_arr.shape == (256,)
-                assert transparency_arr.dtype == 'uint8'
-                transparency = transparency_arr.tobytes()
+
+            assert transparency_arr.shape == (256,)
+            assert transparency_arr.dtype == 'uint8'
+            transparency = transparency_arr.tobytes()
```

Comment:
Yes, I guess these assert might as well apply to both, colormaps retrieved by name, and those user-defined.